openadapt/visualize.py

```diff
@@ -317,7 +317,11 @@ def main() -> None:
     logger.info(f"{fname_out=}")
     output_file(fname_out, title=title)
 
-    # result = show(layout(rows,))
+    result = show(  # noqa: F841
```

Comment:
Can you please clarify what the pragma statement is for?

Reply:
@abrichr The error thrown from flake8 is F841
> [Local variable name is assigned to but never used (F841)](https://www.flake8rules.com/rules/F841.html)